flixel/util/FlxPath.hx

```diff
@@ -135,16 +166,59 @@ class FlxPath implements IFlxDestroyable
 		return this;
 	}
 	
-	public function start(Nodes:Array<FlxPoint>, Speed:Float = 100, Mode:Int = FlxPath.FORWARD, AutoRotate:Bool = false):FlxPath
+	/**
+	 * Starts movement along specified path.
+	 * 
+	 * @param	Nodes			An optional array of path waypoints. If null then previously added points will be used.
+	 * @param	Speed			The speed at which the object is moving on the path.
+	 * @param	Mode			Path following behavior (like looping, horizontal only, etc).
+	 * @param	AutoRotate		Whether the object's angle should be adjusted to the path angle during path follow behavior.
+	 * 
+	 * @return	This path object.
+	 */
+	public function start(?Nodes:Array<FlxPoint>, Speed:Float = 100, Mode:Int = FlxPath.FORWARD, AutoRotate:Bool = false):FlxPath
+	{
+		nodes = (Nodes != null) ? Nodes : nodes;
+		setPathProperties(Speed, Mode, AutoRotate);
+		restart();
+		return this;
+	}
+	
+	/**
+	 * Just sets nodes for this path, plus returns this path object. Could be useful for chaining.
```

Comment:
>Could be useful for chaining.

With that logic, we would need a setter method for every property... I don't really see a good reason to have this function.